### PR TITLE
.gitattributes: ignore assembly tests

### PR DESCRIPTION
**Pull request by eupn**
This should remove assembly test files from language stats leaving the project's main language as JavaScript.

### CODE REVIEW

**Comment by mattgodbolt** [comment]
Oh! Good point; thank you! We should probably ignore `test/demangle-cases` and `test/filters-cases` too (or if we can `test/*-cases/*`)

**Comment by partouf** [comment]
Thanks! No idea when/if this will be processed, but will keep an eye on it

File: .gitattributes [hunk 1]
```diff
@@ -1,4 +1,4 @@
 docs/*  linguist-documentation
 *.s linguist-generated
 lib/handlers/asm-docs.js linguist-generated
-test/cases/* linguist-generated
+test/* linguist-generated
```
Comment (mattgodbolt) [inline .gitattributes]: This of course drops our legitimate javascript tests in our counts, which seems a little unfair.

Comment (eupn) [reply .gitattributes]: I've changed that to `test/*-cases/*`.

Comment (mattgodbolt) [reply .gitattributes]: Thanks! and thanks again for taking the time to improve Compiler Explorer!

**Pull request merged**